### DISCUSSION TITLE
Tillgång till pressrummet.svt.se

### DISCUSSION THREAD

**Discussion by botanicblow** [stefansundin/privatkopiera#102] [Ideas]
Önskar att privatkopiera kan fungera på http://pressrummet.svt.se. Går att registrera sig gratis för att få inloggning. 

Ex. https://pressrummet.svt.se/program/1401892
Tidigare kunde man ta id:t för filmen (32739447) och slänga in på https://www.svtplay.se/video/32739447 men det fungerar ej längre.

**Comment by stefansundin** [comment]
Hej!

Ursäkta att det har tagit evigheter för mig att se detta. Jag förstår dock inte, vad är http://pre**rummet.svt.se? Varför är det censurerat?

**Comment by botanicblow** [comment]
Tjenare Stefan,

ursäktar att det skrevs med två asterixer. Det är pressrummet.svt.se som är
relevant. Jag har inloggning och allting till det men vi brukar få problem
på jobbet när vi skall ladda ned nått från pressrummet. Just nu
skärminspelar vi men det är inte optimalt.
______________________________________

Peter Sundberg
Tfn : 0736 - 850 455
E-post : ***@***.***
______________________________________
